### ISSUE TITLE
PPT image alt text

### ISSUE THREAD

**Issue by emilysaeli** [Microsoft/MCW-OSS-DevOps#9]
Slide 16- image lacks alt text
Slide 17- both images need to be flattened and alt text added
Slide 23- Image needs to be flattened and alt text added (it's nearly impossible to do a screen grab of the pure image under current circumstances)
Slide 24- Image needs to be flattened and alt text added (it's nearly impossible to do a screen grab of the pure image under current circumstances)
